### ISSUE TITLE
sdm-apt-cacher - tiny correction for a comment

### ISSUE THREAD

**Issue by tinker2much** [gitbls/sdm#121]
I set up my new apt-cacher-ng server on bookworm using sdm-apt-cacher, worked great, thank you. 

I noticed this comment block in the source - I believe you forgot the word "systemctl" in the first line:

```
# sudo stop apt-cacher-ng
# sudo rm -rf /var/cache/apt-cacher-ng
# sudo mkdir -p /var/cache/apt-cacher-ng
# sudo chown -R apt-cacher-ng:apt-cacher-ng /var/cache/apt-cacher-ng
# sudo systemctl start apt-cacher-ng
```

**Comment by gitbls** [comment]
oops! Thanks...will fix for next update

**Comment by gitbls** [comment]
Corrected in V9.6
